sandboxed_containers/upgrade-sandboxed-containers.adoc

```diff
@@ -0,0 +1,25 @@
+[id="upgrade-sandboxed-containers"]
+= Upgrade {sandboxed-containers-first}
+include::modules/common-attributes.adoc[]
+:context: upgrade-sandboxed-containers
+
+toc::[]
+
+You can upgrade the components of {sandboxed-containers-first} by upgrading {sandboxed-containers-operator} and {sandboxed-containers-first} artifacts.
+
+[id="sandboxed-containers-upgrade-operator"]
+== Upgrade {sandboxed-containers-operator}
+
+You can use the OpenShift Lifecycle Manager (OLM) to manually or automatically upgrade the {sandboxed-containers-operator}. You can select manual or automatic upgrade during the initial deployment. In the context of manual upgrades, the web console shows the available updates which can be installed by the cluster administrator.
```

Comment:
```suggestion
You can use OpenShift Lifecycle Manager (OLM) to manually or automatically upgrade the {sandboxed-containers-operator}. You can select manual or automatic upgrade during the initial deployment. In the context of manual upgrades, the web console shows the available updates that can be installed by the cluster administrator.
```
A while back, it was decided to refer to "OLM" vs. "the OLM" in cases such as this. (I'm looking to add this to the OpenShift Docs Terms Glossary in this [PR](https://github.com/openshift/openshift-docs/pull/32760).)

ISG:
**that** _conj_
Use “that,” without a comma, to introduce a restrictive clause. Use “which,” preceded by a comma, to
introduce a nonrestrictive clause.